src/cmds/open.ts

```diff
@@ -21,21 +23,36 @@ export default class OpenCommand extends Command {
     this.cmdCategory = CommandCategories.UTILITIES;
     this.position = 1;
 
-    this.args = [];
+    this.args = [
+      {
+        name: 'dash',
+        type: Boolean,
+        description: 'Opens the current project dashboard.',
+      },
+    ];
   }
 
   async run(opts: CommandOptions<Options>) {
     await super.run(opts);
 
+    const { dash } = opts;
     const project = configStore.get('project');
     Command.debug(`project: ${project}`);
 
     if (!project) {
       return Promise.reject(new Error(`Please login using \`${config.get('cli')} login\`.`));
     }
 
-    const hubURL: string = config.get('hub');
-    const url = hubURL.replace('{project}', project);
+    let url: string;
+
+    if (dash) {
+      const selectedVersion = await getProjectVersion(undefined, configStore.get('apiKey'), true);
```

Comment:
I think we should add a little check here to make sure the `apiKey` config value exists:
```suggestion
      const key = configStore.get('apiKey');
      if (!key) {
        return Promise.reject(new Error(`Please login using \`${config.get('cli')} login\`.`));
      }
      const selectedVersion = await getProjectVersion(undefined, key, true);
```
Could you also add a test for this?